test/unit/sqs/index.test.ts

```diff
@@ -301,45 +296,45 @@ describe("SqsQueue tests", () => {
 					done.fail(err);
 				}
 				done();
-				return {retryable: false};
-			})
+				return { retryable: false };
+			});
 
-			await queue.sendMessage(testPayload);
+			queue.sendMessage(testPayload);
 		});
 
-		it("Receive Count and Max Attempts are populated correctly", async (done: DoneCallback) => {
+		it("Receive Count and Max Attempts are populated correctly", (done: jest.DoneCallback) => {
 
 			const testPayload = generatePayload();
-			const receiveCounter = {receivesCounter: 0};
+			const receiveCounter = { receivesCounter: 0 };
 
 			mockRequestHandler.mockImplementation(async (context: Context<TestMessage>) => {
 				/* eslint-disable jest/no-conditional-expect */
 				try {
 					if (receiveCounter.receivesCounter == 0) {
-						expect(context.receiveCount).toBe(1)
-						expect(context.lastAttempt).toBe(false)
+						expect(context.receiveCount).toBe(1);
+						expect(context.lastAttempt).toBe(false);
 					} else if (receiveCounter.receivesCounter == 1) {
-						expect(context.receiveCount).toBe(2)
-						expect(context.lastAttempt).toBe(false)
+						expect(context.receiveCount).toBe(2);
+						expect(context.lastAttempt).toBe(false);
 					} else if (receiveCounter.receivesCounter == 2) {
-						expect(context.receiveCount).toBe(3)
-						expect(context.lastAttempt).toBe(true)
+						expect(context.receiveCount).toBe(3);
+						expect(context.lastAttempt).toBe(true);
 						done();
 						return;
 					}
-				}	catch(err) {
-					done.fail(err)
+				} catch (err) {
+					done.fail(err);
 				}
 
 				receiveCounter.receivesCounter++;
 				throw new Error("Something bad happened");
 			});
 
 			mockErrorHandler.mockImplementation((_error: Error, _context: Context<TestMessage>): ErrorHandlingResult => {
-				return {retryable: receiveCounter.receivesCounter < 3, retryDelaySec: 0, isFailure: true}
-			})
+				return { retryable: receiveCounter.receivesCounter < 3, retryDelaySec: 0, isFailure: true };
+			});
 
-			await queue.sendMessage(testPayload);
+			queue.sendMessage(testPayload);
```

Comment:
Why "await" was removed?

Reply:
I believe we should keep all these awaits in case if unexpected execption happen in sendMessage method though unlikeny. 

However I can see we verify test asynchronously, so this is /nit

Reply:
You can't do both a done callback and an async function as they step on each other's toes.  If something does go wrong, it'll timeout and error after 5 seconds.

Reply:
Ah makes sense now